listenbrainz/mbid_mapping/mapping/mb_metadata_cache.py

```diff
@@ -396,51 +399,223 @@ def config_postgres_join_limit(self, curs):
         curs.execute('SET from_collapse_limit = 15')
         curs.execute('SET join_collapse_limit = 15')
 
-    def mark_rows_as_dirty(self, recording_mbids: List[uuid.UUID], artist_mbids: List[uuid.UUID], release_mbids: List[uuid.UUID]):
-        """Mark rows as dirty if the row is for a given recording mbid or if it's by a given artist mbid, or is from a given release mbid"""
+    def query_last_updated_items(self, timestamp):
+        # there queries here try to mirror the structure and logic of the main cache building queries
+        # note that the tags queries in any of these omit the count > 0 clause because possible removal
+        # of a tag is also a change.
+        # the last_updated considered and last_updated ignored columns below together list all the last_updated
+        # columns a given CTE touches. any other tables touched by a given CTE do not have a last_updated column.
+
+        # further note that these queries only take updates and deletions into consideration, not deletes
```

Comment:
```suggestion
        # these queries only take updates and deletions into consideration, not deletes
```

Reply:
something about _why_ not deletes? because the data is no longer in the db, so we're unable to identify this case?